src/index.ts

```diff
@@ -36,5 +35,4 @@ export {
   ui,
   location,
   app,
-  data,
```

Comment:
Why did you remove this?

Reply:
Typescript does not like to publish the Entity class via an export default since we built the constructor function "solution". I simply dont know xD Also unable to ignore it or expect an error.

Reply:
This is the solution: https://github.com/shopware/admin-extension-sdk/pull/22/files#diff-46a6fe9a2a5e2561f258a496b2df29d3c9306ae1fde832a0f99ac55f8bb8eedfR6 xD